packages/teams-js/src/public/liveShareHost.ts

```diff
@@ -132,6 +132,19 @@ export interface IFluidTenantInfo {
   storageEndpoint: string;
 }
 
+/**
+ * @hidden
+ * Returned from `LiveShareHost.getClientInfo()` to specify the client info for a
+ * particular client in a Live Share session.
+ *
+ * @beta
+ */
+export interface IClientInfo {
+  userId: string;
+  roles: UserMeetingRole[];
+  displayName?: string;
+}
+
 /**
  * Live Share host implementation for O365 and Teams clients.
```

Comment:
This should also have the @hidden keyword like the rest of the things in this file